integration/e2e/scenario_test.go

```diff
@@ -1,5 +1,3 @@
-// +build integration
```

Comment:
Well, my point is that it's not an integration test so technically this tag is wrong.

What if change this tag to `requires_docker`? (:

Reply:
Did that (:

Reply:
I don't care too much how it's called, but feel like this is change just for the change. What if we have integration tests that don't use docker in the future? README.md is not updated.

Reply:
Could you also update `integration/README.md` accordingly?

Reply:
Idea behind using a tag here was to avoid running tests in integration dir when doing `go test ./...` from main Cortex package. I simply used tag name that matched the directory name.

Reply:
> but feel like this is change just for the change

Just seen this and... what? What do you mean? Why I would put on myself more work on myself to improve something in Cortex for no reason? @pstibrany Sorry, I don't get it.

Reply:
![](https://media3.giphy.com/media/a0FuPjiLZev4c/giphy.gif)

Reply:
I just don't feel that renaming the tag was necessary, but ... whatever works. I don't care too much about the name as long as there is a tag in the first place.